app/utils/CurrentDateTime.scala

```diff
@@ -16,11 +16,17 @@
 
 package utils
 
+import java.time.{LocalDateTime, OffsetDateTime}
+import java.time.format.DateTimeFormatter
+
 import javax.inject.{Inject, Singleton}
-import org.joda.time.{DateTime, DateTimeZone}
 
 @Singleton
 class CurrentDateTime @Inject()() {
-
-  def getDateTime: DateTime = DateTime.now(DateTimeZone.UTC)
+  def getDateTime: OffsetDateTime = OffsetDateTime.now()
```

Comment:
This is used for the submission timestamp. Is that okay to be an OffsetDateTime?